listings/applications/components/src/ownable.cairo

```diff
@@ -1,11 +1,16 @@
 use starknet::ContractAddress;
 
 #[starknet::interface]
+// ANCHOR: interface
```

Comment:
Same as before, both the interface and the component should be wrapped within the same anchor, as they are displayed together (see chapter [Ownable](https://starknet-by-example.voyager.online/components/ownable.html#ownable)).
E.g. you can just use a single anchor `// ANCHOR: component`.